### DISCUSSION TITLE
Important problem

### DISCUSSION THREAD

**Discussion by depdonglao** [maxrave-dev/SimpMusic#649] [General]
After new update, i cant download any song
I reinstall app but it still doesnt work
Pls fix it

**Comment by harsh3224h** [comment]


**Reply by depdonglao** [reply]
Nah

**Comment by maxrave-dev** [comment]
Yes, I see error, YouTube change algorithm now

**Reply by mreditss00-svg** [reply]
How can i solve this problem 
I had same prob download failed + can't play any songs
![Screenshot_20250825-154450](https://github.com/user-attachments/assets/acd1f7d4-2907-48b7-b7fb-5df85aab867c)
![Screenshot_20250825-154409](https://github.com/user-attachments/assets/f611bc78-da32-4584-acc8-e9819e88ed1f)
I tried to uninstall and install it many time but the problem isn't solved
